doc/source/installation.md

```diff
@@ -2,7 +2,29 @@
 
 This package depends on GeoPandas, PyGEOS, and Dask.
 
-One way to install all required dependencies is to use the ``conda`` package manager to
+GeoPandas is written in pure Python, but has several dependencies written in C (GEOS, GDAL, PROJ). Those base C libraries can sometimes be a challenge to install. Therefore, we advise you to closely follow the [recommendations](https://geopandas.org/en/stable/getting_started/install.html) to avoid installation problems.
+
+## Easy way
+
+The best way to install Dask-GeoPandas is using `conda` or `mamba` and `conda-forge` channel:
+
+```sh
+conda install -c conda-forge dask-geopandas
+```
+
+## pip
+
+You can install Dask-GeoPandas with `pip` from PyPI but make sure that your environment contains
+properly installed GeoPandas (note that Dask-GeoPandas does not use `fiona` which therefore doesn't
```

Comment:
Maybe point to https://geopandas.org/en/stable/getting_started/install.html#installing-with-pip for the geopandas installation (we indeed don't need fiona here in dask-geopandas, but when doing a `pip install geopandas`, we unfortunately still expect fiona to be installed .. so in practice users will run into the same issues I think)